nuget/AspNetCore.nuspec

```diff
@@ -13,10 +13,15 @@
     <description>ASP.NET Core Module</description>
     <language>en-US</language>
     <tags>Microsoft.AspNetCore.AspNetCoreModule</tags>
+    <contentFiles>
+      <files include="any/any/*/*.dll" buildAction="None" copyToOutput="true" flatten="false" />
+    </contentFiles>
   </metadata>
   <files>
-    <file src="..\artifacts\build\AspNetCore\bin\Release\Win32\aspnetcore.dll" target="runtimes\win7\native\x86\aspnetcore.dll" />
-    <file src="..\artifacts\build\AspNetCore\bin\Release\x64\aspnetcore.dll" target="runtimes\win7\native\x64\aspnetcore.dll" />
+    <file src="..\artifacts\build\AspNetCore\bin\Release\Win32\aspnetcore.dll" target="runtimes\win7\native\aspnetcore_x86.dll" />
```

Comment:
If we were staging this change (which as @Tratcher says, may be unnecessary), this would be wrong wouldn't it? The old code is looking for aspnetcore.dll, not **aspnetcore_x86.dll**.

Reply:
Ah, I think I see. There's a matching change in Hosting (https://github.com/aspnet/Hosting/pull/1268) ? If that's the case, you're not really "staging" this change anyway, so we can just go all the way to the `contentFiles` model.